### DISCUSSION TITLE
目录

### DISCUSSION THREAD

**Discussion by YanWQ-monad** [CollegesChat/university-information#51] [Announcements]
# 这个列表大概率已被弃用

请转到[问卷数据](https://github.com/CollegesChat/university-information/tree/generated)或 [colleges.chat 网站](https://colleges.chat) 以查看更多更全的数据。

<br />

> <del>
>
> ## 一些算不上指导的想法（搬运）
> - [关于高考填报志愿大类招生的一些想法](https://github.com/CollegesChat/university-information/discussions/26)
> - [关于中外合作办学](https://github.com/CollegesChat/university-information/discussions/40)
> 
> ## 数据查看
> 
> </del>
>
> ### 问卷数据
> - GitHub：https://github.com/CollegesChat/university-information/tree/generated
> - 网站：https://colleges.chat
> 
> <del>
>
> ### Discussions & Telegram 数据
> > 此部分尚未合并至“问卷数据”
> 
> - 东南大学
>   - [九龙湖校区](https://github.com/CollegesChat/university-information/discussions/35)
> - [复旦大学](https://github.com/CollegesChat/university-information/discussions/57)
> - 福州大学
>   - [旗山校区](https://github.com/CollegesChat/university-information/discussions/19)
> - 广州大学
>   - [小谷围](https://github.com/CollegesChat/university-information/discussions/22)
> - [吉林大学](https://github.com/CollegesChat/university-information/discussions/9)
> - [清华大学](https://github.com/CollegesChat/university-information/discussions/38)
> - [天津大学](https://github.com/CollegesChat/university-information/discussions/20)
> - 四川大学
>   - [望江校区](https://github.com/CollegesChat/university-information/discussions/6)
>   - [江安校区](https://github.com/CollegesChat/university-information/discussions/5)
> - [深圳大学](https://github.com/CollegesChat/university-information/discussions/59)
> - 厦门大学
>   - [海韵校区](https://github.com/CollegesChat/university-information/discussions/11)
> - [浙江大学](https://github.com/CollegesChat/university-information/discussions/7)
> - [中山大学](https://github.com/CollegesChat/university-information/discussions/8)
> - [北京科技大学](https://github.com/CollegesChat/university-information/discussions/30)
> - 北京理工大学
>   - [良乡](https://github.com/CollegesChat/university-information/discussions/31)
> - 北京师范大学
>   - [珠海校区](https://github.com/CollegesChat/university-information/discussions/56)
> - 北京邮电大学
>   - [沙河校区](https://github.com/CollegesChat/university-information/discussions/2)
> - [长沙理工大学](https://github.com/CollegesChat/university-information/discussions/28)
> - 东北师范大学
>   - [净月校区](https://github.com/CollegesChat/university-information/discussions/82)
> - [福建工程学院](https://github.com/CollegesChat/university-information/discussions/23)
> - 广东工业大学
>   - [大学城校区](https://github.com/CollegesChat/university-information/discussions/25)
> - 华东师范大学
>   - [闵行校区](https://github.com/CollegesChat/university-information/discussions/15)
> - 合肥工业大学
>   - [翡翠湖校区](https://github.com/CollegesChat/university-information/discussions/33)
>   - [宣城校区](https://github.com/CollegesChat/university-information/discussions/37)
> - [华南师范大学](https://github.com/CollegesChat/university-information/discussions/17)（南海/石牌校区）
> - 华中科技大学
>   - [主校区](https://github.com/CollegesChat/university-information/discussions/52)
> - [南京师范大学](https://github.com/CollegesChat/university-information/discussions/21)
> - [南京邮电大学](https://github.com/CollegesChat/university-information/discussions/12)
> - [上海理工大学](https://github.com/CollegesChat/university-information/discussions/27)（申江校区/本部）
> - 四川师范大学
>   - [成龙校区](https://github.com/CollegesChat/university-information/discussions/50)
> - 山东交通学院
>   - [威海新校区](https://github.com/CollegesChat/university-information/discussions/107)
> - [沈阳师范大学](https://github.com/CollegesChat/university-information/discussions/18)
> - 武昌首义学院
>   - [嘉鱼校区](https://github.com/CollegesChat/university-information/discussions/97)
> - 西安交通大学
>   - [兴庆校区](https://github.com/CollegesChat/university-information/discussions/32)
> - 西南财经大学
>   - [柳林](https://github.com/CollegesChat/university-information/discussions/29)
> - 西南政法大学
>   - [渝北校区](https://github.com/CollegesChat/university-information/discussions/55)
> - [中国农业大学](https://github.com/CollegesChat/university-information/discussions/24)
> - 哈尔滨工业大学
>   - [本部](https://github.com/CollegesChat/university-information/discussions/13)
> - [哈尔滨工程大学](https://github.com/CollegesChat/university-information/discussions/3)
> - [西交利物浦大学](https://github.com/CollegesChat/university-information/discussions/58)
> - [北京航空航天大学](https://github.com/CollegesChat/university-information/discussions/14)
> - [对外经济贸易大学](https://github.com/CollegesChat/university-information/discussions/39)
> - [广州城建职业学院](https://github.com/CollegesChat/university-information/discussions/84)
> - [广东第二师范学院](https://github.com/CollegesChat/university-information/discussions/47)
> - [南京航空航天大学](https://github.com/CollegesChat/university-information/discussions/36)（计算机院）
> - [中国科学技术大学](https://github.com/CollegesChat/university-information/discussions/4)
>
> </del>
>

**Comment by fitzhang** [comment]
佛山职业技术学院

**Comment by cfdxkk** [comment]
添加了 [大连东软信息学院 (本科部老校区) ](https://github.com/CollegesChat/university-information/discussions/136)

**Comment by joyhapp** [comment]
呜呜齐齐哈尔医学院

**Comment by yuhouyinren** [comment]
北京交通大学（威海校区）

**Reply by Puke-il** [reply]
其他学校还要等问卷是吗？

**Comment by Long-Zi-Mo** [comment]
安徽理工大学

**Comment by GYFPhoenix** [comment]
中医药大学520左右

**Comment by KAROLLOOOO** [comment]
北京林业大学

**Comment by KAROLLOOOO** [comment]
上海大学

**Comment by KAROLLOOOO** [comment]
北京理工大学

**Comment by Juiceheadc** [comment]
中国刑事警察学院

**Comment by 4466644-hhh** [comment]
有暨南大学的吗

**Comment by gin550** [comment]
填报后没有沈阳工业大学辽阳分校（虽然说我也觉得没人会看这个学校

**Reply by Furlp666** [reply]
有没有肇庆学院的求求了

**Comment by YanWQ-monad** [comment]
Deprecated: 请转到 [问卷数据](https://github.com/CollegesChat/university-information/tree/generated) 或 [colleges.chat 网站](https://colleges.chat) 以查看更多更全的数据。
